openassessment/xblock/static/js/src/lms/oa_staff_area.js

```diff
@@ -125,27 +163,28 @@
          * Upon request, loads the staff grade/assessment section of the staff area.
          * This allows staff grading when staff assessment is a required step.
          *
-         * @param {boolean} clearAndCollapse if true, clear the staff grade form and collapse it. Otherwise
-         *     render the staff grade form if is not already loaded.
          * @returns {promise} A promise representing the successful loading
          * of the staff grade (assessment) section.
          */
-        loadStaffGradeForm: function(clearAndCollapse) {
+        loadStaffGradeForm: function() {
             var view = this;
             var $staffGradeTab = $('.openassessment__staff-grading', this.element);
+            var $staffGradeControl = $staffGradeTab.find('.staff__grade__control');
             var deferred = $.Deferred();
             var showFormError = function(errorMessage) {
                 $staffGradeTab.find('.staff__grade__form--error').text(errorMessage);
             };
 
-            if (clearAndCollapse) {
-                // Collapse the editor and update the counts.
-                $staffGradeTab.find('.staff__grade__control').toggleClass('is--collapsed', true);
-                $staffGradeTab.find('.staff__grade__form').replaceWith('<div class="staff__grade__form"></div>');
-                view.updateStaffGradeCounts();
+            $staffGradeControl.toggleClass('is--showing', true);
+            if (this.staffGradeFormLoaded) {
+                $staffGradeControl.find('.ui-slidable__content').slideDown(
+                    0,  // If transition is visible, the shrinking/growing of the form is seen.
+                    function() {  // This is to work around a JQuery bug (fixed in recent versions of JQuery).
+                        $('.ui-slidable__content').css('overflow', 'visible');
```

Comment:
@talbs There is a bug in "older versions of JQuery" that overflow:hidden is added on slideUp, and not removed on slideDown. I added this workaround because I was seeing the panel appearing at about half-width after the first slide-up.

However, I am not experiencing sizing issues in the other places where I am using slideUp/slideDown (despite the "bad" overflow value existing there), which makes me think there is probably something I could do to a parent container that would cause overflow:hidden to not be an issue. Do you have an idea of what I could do?

This UI is the "Grade Available Responses" staff tools tab at the end of the ORA problem on https://finalgrade.sandbox.edx.org/courses/course-v1:edx+ORA203+course/courseware/8d9584d242b44343bc270ea5ef04ab03/0b0dcc728abe45138c650732af178afb/
The slideUp/slideDown calls happen when you press the "Staff Assessment" button.